ethcore/blockchain/Cargo.toml

```diff
@@ -13,21 +13,23 @@ blooms-db = { path = "../../util/blooms-db" }
 common-types = { path = "../types" }
 ethcore-db = { path = "../db" }
 ethereum-types = "0.6.0"
+keccak-hash = "0.2.0"
 parity-util-mem = "0.1"
 itertools = "0.5"
 kvdb = "0.1"
 log = "0.4"
 parity-bytes = "0.1"
 parking_lot = "0.8"
+rand = "0.6"
```

Comment:
should be it be a dev dep?

Reply:
No, actually the `generator.rs` module is not for tests only